monitoring_tools.py

```diff
@@ -95,6 +100,24 @@ def create_summary_term_list(self, value_list):
                 terms += f"<li>{term}: {value_list[index]}</li>\n"
             return terms
 
+    def append_monitoring_dict(self, monitoring_dict, unique_id, original_path, success):
```

Comment:
new function, that handles the job of checking for existing image ID and paths. Allows for appending several image paths to the same ID